cime_config/tests.py

```diff
@@ -655,8 +655,12 @@
 
     "e3sm_scream_v1_dp-eamxx" : {
         "time"  : "01:00:00",
+        # each test 225 phys cols, roughly size of ne2
         "tests" : (
-            "ERS_P16_Ln22.ne30_ne30.F2010-SCREAMv1-DP-DYCOMSrf01", # 225 phys cols, roughly size of ne2
+            "ERS_P16_Ln22.ne30_ne30.F2010-SCREAMv1-DP-General.scream-dpxx-dycomsrf01",
+            "ERS_P16_Ln22.ne30_ne30.F2010-SCREAMv1-DP-General.scream-dpxx-arm97",
+	    "ERS_P16_Ln22.ne30_ne30.F2010-SCREAMv1-DP-General.scream-dpxx-comble",
```

Comment:
I know this is nitpicky, but if you change "General" to "OBS" then it would line up with "RCE"!

Reply:
Yeah, I never liked the name "General" but was feeling uninspired to think of anything better.... but your suggestion inspired me.  I'll likely change it to "IOP" which is much more descriptive (and will line up).